docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -2,67 +2,59 @@
+Make sure that all Pod Security Policies have been migrated, which should result in equivalent security controls still being enforced (e.g. [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/), [Gatekeeper](https://github.com/open-policy-agent/gatekeeper), [Kubewarden](https://www.kubewarden.io/), [Kyverno](https://kyverno.io/), [NeuVector](https://neuvector.com/)) and all PodSecurityPolicy objects removed from the cluster.
 
 :::info important
 The new policy enforcement must be added before the PodSecurityPolicy objects are removed, or you risk creating a window of opportunity for privilege escalation within the cluster.
-
 :::
 
 ## Pod Security Admission Configuration Templates
 
 Rancher comes with two Pod Security Admission Configuration (PSAC) templates that you can assign to a cluster:
-- `rancher-privileged`: This is the most permissive configuration and doesn't restrict the behavior of any pod.
-- `rancher-restricted`: This policy follows current pod-hardening best practices.
-
+- `rancher-privileged`: This is the most permissive configuration and does not restrict the behavior of any pod, allowing for known privilege escalations. This policy contains no exemptions.
+- `rancher-restricted`: This policy defines a heavily restricted configuration, based on current pod hardening best practices. This policy contains namespace level exemptions for Rancher components, as described below.
 
 If you are a Rancher administrator or have restricted administrator privileges, you can customize restrictions and permissions by creating additional PSAC templates, or by editing existing templates.
 
 :::caution
-
 If you edit an existing PSAC template while it is still in use, it *will* affect all clusters that have been assigned that template.
-
 :::
 
-If you want to allow users other than the Rancher administrator to manage PSAC templates, you can bind the user to a role that grants the user all verbs ("*") on `management.cattle.io/podsecurityadmissionconfigurationtemplates`.
+If you want to allow users other than the Rancher administrator to manage PSAC templates, you can bind the user to a role that grants the user all verbs (`"*"`) on `management.cattle.io/podsecurityadmissionconfigurationtemplates`.
 
 :::warning
-
 Any user that is bound to the above permission will be able to change the restriction levels on all managed clusters which use this template, including ones they have no permissions on.
-
 :::
 
 
-### Rancher On PSA-restricted Clusters
+### Rancher on PSA-restricted Clusters
+
+When you run Rancher on a Kubernetes cluster that enforces a restrictive PSS policy by default, such as the `rancher-restricted` PSS policy, you will need to exempt the following namespaces, otherwise the policy might prevent Rancher system pods from properly running.
```

Comment:
There's some slippage here between PSA and PSS, as the heading uses "PSA-restricted" but the body text talks of PSS policies. They're closely related concepts with similar acronyms, and a reader might get confused on which is which. Maybe we can avoid the acronyms here by using "security policy"? Example: "[...] a restrictive security policy by default, such as the `rancher-restricted` policy".